src/Moryx.Orders.Web/src/app/dialogs/create-dialog/create-dialog.component.ts

```diff
@@ -217,8 +225,12 @@ export class CreateDialogComponent implements OnInit {
         onlySelf: true,
       });
     }
-  }
 
+    this.operationNumberFormControl.reset('');
+    this.operationNumberFormControl.markAsPristine();
```

Comment:
this and the following line should be redundant, see https://angular.dev/api/forms/FormControl -> reset

Also check line 208, that shouldn't be necessary anymore either